.gitignore

```diff
@@ -59,3 +59,8 @@ pip-wheel-metadata/
 
 # Mac OSX
 .DS_Store
+
+# python -m venv
+lib
+lib64
```

Comment:
Can these be more specific? Particularly `lib` because sometimes a package may have a directory that is actually named `lib`.

Reply:
running `python -m venv .` generates a set of folders,
* bin
* lib
* include
* lib64 -> lib ( symlink )